### PR TITLE
Add WhyLabs Writer example notebook

### PR DESCRIPTION
**Pull request by FelipeAdachi**
- Add WhyLabsWriter example notebook
- changing both writer examples into a `writer` folder under `integrations`
- updating READMES and other markdown files in documentation

## Description

This repository needs 

The commits in this pull request will 

## Changes

- Imperative commit or change title (commit ID) 
  - Add some bullet points to explain the change
- Next commit or change (commit ID)

## Related

Relates to organization/repo#number

Closes organization/repo#number

- [ ] I have reviewed the [Guidelines for Contributing](CONTRIBUTING.md) and the [Code of Conduct](CODE_OF_CONDUCT.md).

### CODE REVIEW

File: python/examples/integrations/writers/Writing_to_WhyLabs.ipynb [hunk 1]
```diff
@@ -0,0 +1,482 @@
...
+    "In this example, we'll show how to send your logged profiles to your monitoring dashboard at WhyLabs Platform.\n",
+    "We will:\n",
+    "\n",
+    "- Define environment variables with the appropriate Credentials and IDs\n",
+    "- Log data into a profile\n",
+    "- Use the WhyLabs Writer to send the profile to your Project at WhyLabs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
```
Comment (jamie256) [inline python/examples/integrations/writers/Writing_to_WhyLabs.ipynb:30]: We might want to include a pip install command to pull in the whylabs extra?

```
!pip install whylogs[whylabs]
```

Comment (FelipeAdachi) [reply python/examples/integrations/writers/Writing_to_WhyLabs.ipynb:30]: Yes, forgot about that.

Comment (FelipeAdachi) [reply python/examples/integrations/writers/Writing_to_WhyLabs.ipynb:30]: Added in the last commit.

**Review by TheMellyBee — APPROVED** [review]
This looks great, and I ran through it to do the example for Flask. :)

**Pull request merged**